src/models/nftRewardTier.ts

```diff
@@ -31,7 +31,7 @@ export type DelegateVersion = JB721DELEGATE_V1 | JB721DELEGATE_V1_1
 
 // How the reward tiers are stored in the contracts
 export type JB721TierParams = {
-  id?: BigNumber //undefined for outgoing tier (in launch or adjustTiers tx)
+  id: BigNumber //undefined for outgoing tier (in launch or adjustTiers tx)
   remainingQuantity?: BigNumber //undefined for outgoing tier (in launch or adjustTiers tx)
```

Comment:
**suggestion (non-blocking)** ideally we might have a separate type that handles this case. As in, JB721TierParams might have `remainingQuantity`, but then some other type would be `type EditingJB721TierParams = Omit<JB721TierParams, 'remainingQuantity'>`, for example

Reply:
Yeah this is a good idea, although will be a solid diff. Happy to get this PR in now since it's already quite hefty?